toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -1,5 +1,5 @@
 filesystem-1.1-7.cm1.x86_64.rpm
-kernel-headers-5.10.32.1-4.cm1.noarch.rpm
+kernel-headers-5.10.32.1-3.cm1.noarch.rpm
```

Comment:
Why are there changes for other packages than `curl`?

Reply:
Applies to other files too.

Reply:
needs to be refreshed with 1.0-dev